skatepark/sass/base/_text.scss

```diff
@@ -6,4 +6,18 @@
 p {
 	margin-top: calc( 0.5 * var(--wp--custom--margin--vertical) );
 	margin-bottom: calc( 0.5 * var(--wp--custom--margin--vertical) );
-} 
\ No newline at end of file
+}
+
+.wp-block-post-content {
+	p:first-of-type {
+		text-indent: 15%;
+	}
+}
+
+.wp-block-post-author__content,
+.wp-block-post-comments,
+.wp-block-quote {
```

Comment:
Done

Reply:
> having the post comments block inside wp-block-post-content would be weird, would it even work?

It would be weird but it does work. There could be cases where you wanted to encourage comments so you put the form into the post content? If you let people do it, someone will find a use for it!